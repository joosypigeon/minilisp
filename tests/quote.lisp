(define quoted-list
  (list
    '(a (b c) d)
    'hello
    '(a b c)
    'a
    '(1 2 3)
    '((a b) c)))

(define loop-print
  (lambda (xs)
    (if (equal? xs NIL)
        NIL
        (begin
          (print (car xs))
          (loop-print (cdr xs))))))
          
(loop-print quoted-list)