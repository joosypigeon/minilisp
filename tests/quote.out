(a (b c) d)
hello
(a b c)
a
(1 2 3)
((a b) c)
